contrib/smt-to-rmt-helpers/rmt-data-import

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env ruby
+
+# FIXME: argument parsing with getopt? To produce help and such.
+unless (ARGV[0])
+  warn "Usage: #{$PROGRAM_NAME} /path/to/smt-data-dir"
+  warn ''
+  warn 'Please specify a path to a directory with an unpacked SMT data tarball as the first argument.'
+  exit 1
+end
+
+data_dir = ARGV[0]
+rmt_path = ENV['RMT_PATH'] ? ENV['RMT_PATH'] : '/usr/share/rmt/' # for testing in dev-mode
+
+require_relative "#{rmt_path}/config/boot"
+$LOAD_PATH.unshift File.join(rmt_path, 'lib')
+
+require 'active_support'
+require 'active_record'
+require 'erb'
+require 'yaml'
+require 'rmt/config'
+require 'csv'
+
+relative_load_paths = %w[lib lib/rmt app/models app/services].map { |dir| File.join(rmt_path, dir) }
+ActiveSupport::Dependencies.autoload_paths += relative_load_paths
+
+db_config = RMT::Config.db_config
+ActiveRecord::Base.establish_connection(db_config)
+
+#####
+
+# FIXME: need to check that products table isn't empty
+# FIXME wrap everything into a gigantic transaction?
```

Comment:
I prefer the logic broken down into methods tbh. The bash script actually looks cleaner to me :see_no_evil: